packages/dcos-metrics-collector/build

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Alternately if credentials arent needed to poll mesos agents then this is a moot point.